src/include/packager/repositories_include.rb

```diff
@@ -497,5 +461,22 @@ def EditDialog
       ret = SourceDialogs.EditDialog
       ret
     end
+
+    # Evaluate the default autorefresh flag for the given repository URL.
+    # @param url [String] Repository URL
+    # @return [Boolean] The default autorefresh flag for the URL
+    def autorefresh(url)
```

Comment:
quite unfamiliar name, I expect it set autorefresh for given url. What about `autorefresh?` or `use_autorefresh_for?`